crates/duty-tracker/src/predicates.rs

```diff
@@ -43,30 +43,34 @@ pub(crate) fn deposit_request_info(
     pegout_graph_params: &PegOutGraphParams,
     stake_index: u32,
 ) -> Option<DepositInfo> {
+    const X_ONLY_PUBKEY_SIZE: usize = 32;
+
     let deposit_request_output = tx.output.first()?;
     if deposit_request_output.value <= pegout_graph_params.deposit_amount {
         return None;
     }
     // TODO(proofofkeags): validate that the script_pubkey pays to the right operator set
 
     let ee_address_size = sidesystem_params.ee_addr_size;
-    let (take_back_leaf_hash, el_addr) =
+    let (recovery_x_only_pk, el_addr) =
         magic_tagged_data(MAGIC_BYTES, &tx.output.get(1)?.script_pubkey).and_then(|meta| {
             if meta.len() != TAPROOT_CONTROL_NODE_SIZE + ee_address_size {
```

Comment:
Since we are no longer including the tapleaf hash, we should update this condition too.
```suggestion
            if meta.len() != SCHNORR_PUBLIC_KEY_SIZE + ee_address_size {
```

Reply:
Done in 42dbcc2dfec1013af435fdbd9b2936af2fed010d

Thanks!